instructions.json

```diff
@@ -18,25 +18,35 @@
             "defaultInstruction": "Head {modifier}[ on {way_name}]"
         },
         "end of road": {
-            "defaultInstruction": "Turn {modifier}[ onto {way_name}]"
+            "defaultInstruction": "Turn {modifier}[ onto {way_name}] at the end of the road",
```

Comment:
Well. It is helpful if you have a long way to the intersection. So I feel what I would prefer is to have a long version for announcing what comes next and a short version that would omit this additional information at the turn.

So if you make the turn, you could get an immediate information `at the end of the road, turn left`. And when you are at the intersection `turn left`. This at the end-of-the-road would replace the distance in the announcement.

I imagine: `Turn left at the end of the road` instead of `In 200 meters, turn left at the end of the road`.